flake.nix

```diff
@@ -118,23 +116,18 @@
           in
           {
             build01 = nixosSystem {
-              system = "x86_64-linux";
               modules = [ ./hosts/build01/configuration.nix ];
             };
             build02 = nixosSystem {
-              system = "x86_64-linux";
               modules = [ ./hosts/build02/configuration.nix ];
             };
             build03 = nixosSystem {
-              system = "x86_64-linux";
               modules = [ ./hosts/build03/configuration.nix ];
             };
             build04 = nixosSystem {
-              system = "aarch64-linux";
               modules = [ ./hosts/build04/configuration.nix ];
             };
             web01 = nixosSystem {
```

Comment:
I believe we save evaluation time if we set pkgs directly because then nixpkgs is not re-imported for every system: https://github.com/Mic92/dotfiles/blob/05c0dbc5ba882fbe4d51cffe6e124cdfcd924983/nixos/flake-module.nix#L118

Reply:
Would have to benchmark that but if I had to guess a few 100MB per machine.

Reply:
Not so sure how much we safe for the infra repository:

https://github.com/TUM-DSE/doctor-cluster-config/pull/322

This repo has significantly more hosts.

Reply:
Everything is buildable but hercules doesn't seem to like `pkgs`, is there something else we need to do for this?

Reply:
@Mic92 Any suggestions for fixing this?

Reply:
Looks like it's fixed? Maybe unrelated?

Reply:
ok. no, it came back.